Unable to build crate when git is not installed

Error occurs when building xmp-toolkit-rs without git installed.

```
error: failed to run custom build command for `xmp_toolkit v1.0.0`

Caused by:
  process didn't exit successfully: `.../build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=build.rs
  > git submodule init


  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/xmp_toolkit-1.0.0/build.rs:369:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:59: build-release-linux] Error 101
```

--- a/build.rs
+++ b/build.rs
@@ -375,22 +375,17 @@ where
     I: IntoIterator<Item = S>,
     S: AsRef<OsStr>,
 {
-    let output = std::process::Command::new("git")
-        .args(args)
-        .output()
-        .unwrap();
-
-    println!(
-        "--- stdout ---\n{}\n\n--- stderr ---\n{}\n\n",
-        String::from_utf8(output.stdout).unwrap(),
-        String::from_utf8(output.stderr).unwrap()
-    );
-
-    // When we run inside the docs.rs environment (and, presumably,
-    // any client that is building xmp-toolkit-rs as a dependency),
-    // the submodule doesn't exist, so we should ignore any
-    // error from git.
-    // assert_eq!(output.status.code().unwrap(), 0);
+    if let Ok(output) = std::process::Command::new("git").args(args).output() {
+        println!(
+            "--- stdout ---\n{}\n\n--- stderr ---\n{}\n\n",
+            String::from_utf8(output.stdout).unwrap(),
+            String::from_utf8(output.stderr).unwrap()
+        );
+    } else {
+        eprintln!("INFO: git command failed");
+        eprintln!("  If building from crates.io, this should be OK.");
+        eprintln!("  Otherwise, please manually ensure that submodules are up to date.");
+    }
 }
 
 fn compile_for_docs() {

